packages/manager/apps/dedicated/client/app/account/user/components/newAccountForm/new-account-form-controller.js

```diff
@@ -130,7 +142,11 @@ export default class NewAccountFormController {
             editedRule.hasBottomMargin = this.coreConfig.isRegion('US');
           } else {
             editedRule.readonly = this.readonly.includes(editedRule.fieldName);
-            editedRule.hasBottomMargin = true;
+            editedRule.hasBottomMargin =
+              FIELD_WITHOUT_MARGIN_BOTTOM.indexOf(editedRule.fieldName) === -1;
+            if (editedRule.fieldName === 'phone') {
```

Comment:
in this short block, you use 3 different methods `includes`, `indexOf` and `===`. You should use only one to keep the code more readable and consistent (`includes` for example)